ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5696,6 +5698,419 @@ kong_apis:
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: getUserDetailsOfDiscussion
+    uris: "{{ discussions_prefix }}/user"
+    upstream_url: "{{ discussions_mw_service_url }}/user"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: getCategoryDetailsOfDiscussion
+    uris: "{{ discussions_prefix }}/category"
+    upstream_url: "{{ discussions_mw_service_url }}/category"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: getUnreadTopicsOfDiscussion
+    uris: "{{ discussions_prefix }}/unread"
+    upstream_url: "{{ discussions_mw_service_url }}/unread"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: getRecentTopicsOfDiscussion
+    uris: "{{ discussions_prefix }}/recent"
+    upstream_url: "{{ discussions_mw_service_url }}/recent"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: getPopularTopicsOfDiscussion
+    uris: "{{ discussions_prefix }}/popular"
+    upstream_url: "{{ discussions_mw_service_url }}/popular"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: getTopTopicsOfDiscussion
```

Comment:
@VenkatKadiveti do check this,  it has indentation issues, below APIs are also having the same issues.